scripts/set-permissions.ts

```diff
@@ -0,0 +1,73 @@
+import { deriveAirnodeXpub, deriveSponsorWalletAddress } from '@api3/airnode-admin';
+import { AirnodeRrp, RrpBeaconServer } from '@api3/airnode-protocol';
+import { ethers, Wallet } from 'ethers';
+import * as node from '@api3/airnode-node';
+import {
+  cliPrint,
+  deriveKeeperSponsorWallet,
+  fundAWallet,
+  getDeployedContract,
+  getProvider,
+  readIntegrationInfo,
+  runAndHandleErrors,
+} from '../src';
+
+const main = async () => {
+  const provider = getProvider();
+  const integrationInfo = await readIntegrationInfo();
+  const airnodeRootWallet = ethers.Wallet.fromMnemonic(integrationInfo.mnemonic).connect(provider);
+  const airnodeHDNode = ethers.utils.HDNode.fromMnemonic(integrationInfo.mnemonic);
+  const keeperSponsor = new Wallet(airnodeHDNode.derivePath(`m/45'/60'/0'/0/0`)).connect(getProvider());
+
+  const keeperSponsorWallet = deriveKeeperSponsorWallet(airnodeHDNode, keeperSponsor.address, provider).connect(
+    provider
+  );
+
+  const airnodeRrpInstance = (
+    (await getDeployedContract('@api3/airnode-protocol/contracts/rrp/AirnodeRrp.sol')) as AirnodeRrp
+  ).connect(airnodeRootWallet);
+  cliPrint.info(`Deployed AirnodeRrp: ${airnodeRrpInstance.address}`);
+
+  const sponsor = ethers.Wallet.fromMnemonic(integrationInfo.mnemonic).connect(provider);
+  const airnodeXpub = deriveAirnodeXpub(airnodeRootWallet.mnemonic.phrase);
+  cliPrint.info(`Airnode XPub: ${airnodeXpub}`);
+
+  const sponsorWalletAddress = await deriveSponsorWalletAddress(
+    airnodeXpub,
+    airnodeRootWallet.address,
+    sponsor.address
+  );
+
+  const requestSponsorWallet = node.evm.deriveSponsorWallet(airnodeHDNode, sponsorWalletAddress).connect(provider);
+  cliPrint.info(`Keeper Sponsor Wallet: ${keeperSponsorWallet.address}`);
+  cliPrint.info(`Request Sponsor Wallet: ${requestSponsorWallet.address}`);
+
+  await fundAWallet(provider, airnodeRootWallet, sponsorWalletAddress);
+  await fundAWallet(provider, airnodeRootWallet, keeperSponsorWallet.address);
+
+  cliPrint.info(`Now checking if ${keeperSponsorWallet.address} has beacon updater permission...`);
+  const beacon = (
+    await getDeployedContract('@api3/airnode-protocol/contracts/rrp/requesters/RrpBeaconServer.sol')
+  ).connect(sponsor.connect(getProvider())) as RrpBeaconServer;
+
+  if (await beacon.sponsorToUpdateRequesterToPermissionStatus(sponsor.address, keeperSponsorWallet.address)) {
+    cliPrint.info(`Beacon updater already has the updater permission.`);
+    process.exit(0);
+  }
+
+  cliPrint.info(`Giving the beacon updater permission to the request wallet ${keeperSponsorWallet.address}`);
+  const tx = await beacon.setUpdatePermissionStatus(keeperSponsorWallet.address, true);
```

Comment:
🥇 
This is great and fixes item #2 in [this](https://github.com/api3dao/airkeeper/blob/7474ae5f279465ef538f5da894664b0a7e626b5f/src/start.ts#L299) prerequisites list but I think we are still missing #1 or maybe I'm just not able to find it in this guide.

Reply:
Yes, we're definitely still missing step 1, I'll add that shortly.